README.md

```diff
@@ -1,60 +1,40 @@
 # Project Name
 OOP-IN-RUBY
-> Description the project.
-
+> A project created just for learning Object-Oriented-Programming(OOP) in Ruby
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- RUBY
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**instructions on setting up your project locally.**
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
+**Computer**
 ### Setup
-
+**clone this repo `https://github.com/jheart-vic/OOP-IN-RUBY.git`**
 ### Install
 
-### Usage
+**Install Ruby**
 
-### Run tests
+## Author
 
-### Deployment
+👤 **Author**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author** on _line 26_ of your ReadMe with your full name. This will make your ReadMe more professional